src/Sylius/Bundle/AdminBundle/Resources/views/layout.html.twig

```diff
@@ -3,7 +3,7 @@
 {% block title %} | Sylius{% endblock %}
 
 {% block stylesheets %}
-    {% include '@SyliusUi/_stylesheets.html.twig' with {'path': 'assets/admin/css/style.css'} %}
```

Comment:
I could see someone modifying `@SyliusUi/_stylesheets.html.twig` with an intention to include some asset in all the UI. Same applies to `@SyliusUi/_javascripts.html.twig`. If we don't include that file, it would effectively be unused and it would have no effect in the next minor Sylius release.

I can see two solutions:
 - Revert this change (should still work since the path is same)
 - Use the removed code in `@SyliusAdmin/_styles.html.twig` (so that `layout.html.twig` uses `_styles.html.twig` which uses `_stylesheets.html.twig`)

No matter what we decide to go with, it should be done for styles and scripts for both Admin and Shop in the same way.

Reply:
good point, the second solution seems better to me